lib/dependencies.js

```diff
@@ -14,6 +14,7 @@ container.instance("apiServer", require("./api-server"));
 container.register("base64", require("./base64.js"));
 container.singleton("logger", require("./logger"));
 container.instance("middlewareProfiler", require("./middleware-profiler"));
+container.register("otDate", require("./ot-date"));
```

Comment:
The capitalization here and in the test doesn't match.  You should consider if you are testing something that you want to treat as a class (probably not) or something that you want to treat as an object (most likely) and then capitalize or uncapitalize to match.